pipelines.md

```diff
@@ -0,0 +1,323 @@
+---
+permalink: /guides/pipelines-getting-started/
```

Comment:
The change to the permalink threw me when I tried to build this locally. The repo is still `guide-working-with-pipelines` but the permalink has changed to `pipelines-getting-started`, which means that the URL for the guide is https://kabanero.io/guides/pipelines-getting-started/  (even though the title is "Build and deploy applications with pipelines")

This goes against the contribution guidelines and I would recommend that we change the permalink back to `guides/working-with-pipelines`.

Reply:
Agree